Boa.Constrictor/WebDriver/Extensions/AbstractWebLocatorListQuestion.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Boa.Constrictor.Screenplay;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Boa.Constrictor.WebDriver
+{
+    /// <summary>
+    /// Abstract class for any Web questions that return a collection of strings using a Web locator and a property name.
```

Comment:
/// Abstract class for any Web questions that return a collection of strings using a Web locator.